front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java

```diff
@@ -592,15 +581,15 @@ public void doRetry(Closure operation,
                       String action,
                       String resource) {
       gcsSafeRetry.doRetry(operation,
-                           action,
                            resource,
                            null,
-                           null,
                            Arrays.asList(500),
                            null,
                            maxWaitInterval,
                            retryIntervalBase,
                            jitterMultiplier,
-                           maxRetries);
+                           maxRetries,
+                           Collections.singletonMap("action", action),
+                           registry);
```

Comment:
@duftler I think you made some changes here recently. Just want to make sure what I've done here is right.